src/main/webapp/WEB-INF/views/car/add.jsp

```diff
@@ -0,0 +1,29 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Create new car</title>
+</head>
+<body>
+<table>
+    <tr>
+        <td>ID</td>
+        <td>NAME</td>
+        <td>COUNTRY</td>
+    </tr>
+    <c:forEach items="${manufacturers}" var="manufacturer">
+        <tr>
+            <td><c:out value="${manufacturer.id}"/></td>
+            <td><c:out value="${manufacturer.name}"/> </td>
+            <td><c:out value="${manufacturer.country}"/> </td>
+        </tr>
+    </c:forEach>
+</table>
+
+<form method="post" action="${pageContext.request.contextPath}/car/add">
+    Model <input type="text" name="model"><br>
+    ManufacturerId <input type="text" name="manufacturerId"><br>
```

Comment:
let's use a more suitable input type for manufacturer id
https://www.w3schools.com/html/html_form_input_types.asp